packages/pds/src/api/app/bsky/feed/getPostThread.ts

```diff
@@ -22,25 +22,41 @@ import {
   getRepoRev,
   handleReadAfterWrite,
 } from '../util/read-after-write'
+import { authPassthru } from '../../../../../api/com/atproto/admin/util'
```

Comment:
I noticed a build error that might be related to this—is it possible there's one too many `../`?

Reply:
hmm.. no idea how this happened though since it was auto-imported by vscode lol.